### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by evanyang1**
style updates for Login and Landing pages

Please give feedback on how to make pages more beautiful & a e s t h e t i c

### CODE REVIEW

File: client/src/pages/Landing.js [hunk 1]
```diff
@@ -44,6 +47,9 @@ const useStyles = makeStyles({
   errors: {
     color: 'red',
   },
+  alreadyHaveAccount: {
```
Comment (ThinkTink) [inline client/src/pages/Landing.js]: Let's make this name more about what it does then where it's used

File: client/src/pages/Login.js [hunk 2]
```diff
@@ -28,12 +31,15 @@ const useStyles = makeStyles({
   marginBottom50: {
     marginBottom: '50%'
   },
-  marginLeft15: {
-    marginLeft: '15%'
+  createAccount: {
```
Comment (ThinkTink) [inline client/src/pages/Login.js]: same with this name

**Review by ThinkTink — COMMENTED** [review]
We want to merge this into dev, not socket right?
Did you have any specific questions about the UI?

**Comment by ThinkTink** [comment]
Can you merge dev into this branch, then flag me again for review? thanks!

**Comment by ThinkTink** [comment]
Oh I see what the problem is - this is still set to merge into socket instead of dev. Is that intentional?

File: app.js [hunk 3]
```diff
@@ -13,6 +13,24 @@ const { json, urlencoded } = express;
...
+          "Access-Control-Allow-Origin": req.headers.origin, //or the specific origin you want to give access to,
+          "Access-Control-Allow-Credentials": true
+      };
+      res.writeHead(200, headers);
+      res.end();
+  },
+  cors: {
+    origin: "https://example.com",
+    methods: ["GET", "POST"],
+    allowedHeaders: ["my-custom-header"],
```
Comment (ThinkTink) [inline app.js:30]: Looks like some of the socket stuff still wound up here. Let's be sure to remove the template stuff!

Comment (evanyang1) [reply app.js:30]: Interesting, they seem to have disappeared in my end, and i deleted my branch

**Pull request merged**